### DISCUSSION TITLE
-display-cdn option creates issue while toolchaining with httpx

### DISCUSSION THREAD

**Discussion by praveenroot03** [projectdiscovery/naabu#951] [General]
### Current Behavior:
While using the Naabu with the option `-display-cdn` with the `silent` mode created a additional field denoting CDN of the host. 
`host:port [CDN]` 
Due to this behavior while toolchaining  Naabu with httpx fails, because httpx cannot parse the input that contains additional CDN field.

`echo cloudflare.com | naabu -pf top-web-port.txt -verify -sa -o open-ports.txt -c 75 -rate 3000 -display-cdn -silent | httpx -title -status-code -web-server -vhost -screenshot -o http-probe-url.txt -v

[DBG] Store response directory not specified, using "output"
[DBG] Failed '/cloudflare.com:8443 [cloudflare]': unsupported protocol scheme ""
[DBG] Failed '/cloudflare.com:443 [cloudflare]': unsupported protocol scheme ""
[DBG] Failed '/cloudflare.com:80 [cloudflare]': unsupported protocol scheme ""
[DBG] Failed '/cloudflare.com:8080 [cloudflare]': unsupported protocol scheme ""`

### Expected Behavior:
It is expected to only pipe the output of host:port

**Comment by Mzack9999** [comment]
This seems the expected behavior as you are using the `display-cdn` flag. If for any reason these are flags you can't avoid. You can truncate the output by using `sed` within the pipe:
```console
naabu ... | sed 's/\[[^]]*\]//g' | httpx ...
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
